File: ExamApp.Domain/Question.cs
using System;

namespace ExamApp.Domain
{
    public class Question
    {
        public Guid Id { get; set; }
        public string Content { get; set; }
        public Guid TestId { get; set; }
        public string Answer1 { get; set; }
        public string Answer2 { get; set; }
        public string Answer3 { get; set; }
        public string Answer4 { get; set; }
        public int CorrectAnswer { get; set; }
    }
}

File: ExamApp.UI/client-app/src/app/models/question.ts
export interface IQuestion {
    id: string;
    content: string;
    testId: string;
    answer1: string;
    answer2: string;
    answer3: string;
    answer4: string;
    correctAnswer: number;
}

export class QuestionFormValues implements Partial<IQuestion> {
    id?: string = undefined;
    content: string = '';
    testId?: string = undefined;
    answer1: string = '';
    answer2: string = '';
    answer3: string = '';
    answer4: string = '';
    correctAnswer: number = undefined!;

    constructor(init?: IQuestion) {
        Object.assign(this, init);
    }
}